packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port.py

```diff
@@ -159,9 +160,25 @@ async def _set(self, new_value: ItemConcreteValue) -> None:
             # convert the concrete value to a data value
             converted_value: ItemConcreteValue = self._py_value_converter(new_value)
 
-            if isinstance(converted_value, Path):
-                if not converted_value.exists() or not converted_value.is_file():
+            def _raise_if_file_not_on_disk(path: Path) -> None:
+                if not path.exists() or not path.is_file():
+                    log.warning("Could not find a valid file %s", path)
                     raise InvalidItemTypeError(self.property_type, str(new_value))
+
+            if isinstance(converted_value, Path):
+                if converted_value.is_symlink():
```

Comment:
IMO following links should be optional (SEE  discussion about ``follow_symlinks`` in [os](https://docs.python.org/3/library/os.html) ) 

A situation like ``workdir``

```console
$ ls -la workdir
drwxrwxr-x 2 scu scu 4096 Nov 10 22:21 .
drwxrwxr-x 8 scu scu 4096 Nov 10 22:21 ..
-rw-rw-r-- 1 scu scu    0 Nov 10 22:21 file.dat
lrwxrwxrwx 1 scu scu   11 Nov 10 22:21 inputs -> /data/inputs
lrwxrwxrwx 1 crespo crespo   12 Nov 10 22:21 outputs -> /data/outputs
```
I would like to upload to s3 ``workdir`` but avoid ``inputs`` and ``outputs``. But when downloaded, I would like the symlinks to be restored

Reply:
- in nodeports config
- nodeports will ultimately be used almost exclusively in the dy-sidecar
- it is up the the service to decide ``follow_symlinks``
- I would rather default ``follow_symlinks=False`` as in the os lib

Reply:
If you look carefully, `follow_symlinks=False` is Falso only when dealing with permission changes and some other border line cases. When the file needs to be read/written `follow_symlinks=True`.
That is exactly our case. I would default it to `True`
I agree into storing the configuration into nodeports.